### PR TITLE
verify: No phantomjs, no TEST_IMAGES_RHEL

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by stefwalter** [comment]
@mvollmer Have you built a new image and pushed it to the docker hub? That would be the next step to getting this merged.

**Comment by stefwalter** [comment]
Actually I can do the build and push. Since I have some make file changes to test out.

**Comment by mvollmer** [comment]
> Have you built a new image and pushed it to the docker hub?

No.  Do we do that before merging?  How do we test changes to the verify container, actually?

**Comment by stefwalter** [comment]
I tried this out on cockpit-11 and got tons of failures like:

```
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at process._tickCallback (internal/process/next_tick.js:98:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Module.runMain (module.js:606:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at run (bootstrap_node.js:394:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at startup (bootstrap_node.js:149:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at bootstrap_node.js:509:3
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: events.js:160
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: throw er; // Unhandled 'error' event
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: ^
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: This socket is closed
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._writeGeneric (net.js:680:19)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._write (net.js:731:8)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at doWrite (_stream_writable.js:334:12)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at writeOrBuffer (_stream_writable.js:320:5)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.Writable.write (_stream_writable.js:247:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.write (net.js:658:40)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.ondata (_stream_readable.js:555:20)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at emitOne (events.js:96:13)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.emit (events.js:188:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at readableAddChunk (_stream_readable.js:176:18)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Restarting browser...
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error executing phantom at /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: spawn /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs ENOENT
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at exports._errnoException (util.js:1022:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at onErrorNT (internal/child_process.js:359:16)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at process._tickCallback (internal/process/next_tick.js:98:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Module.runMain (module.js:606:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at run (bootstrap_node.js:394:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at startup (bootstrap_node.js:149:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at bootstrap_node.js:509:3
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: events.js:160
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: throw er; // Unhandled 'error' event
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: ^
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: This socket is closed
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._writeGeneric (net.js:680:19)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._write (net.js:731:8)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at doWrite (_stream_writable.js:334:12)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at writeOrBuffer (_stream_writable.js:320:5)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.Writable.write (_stream_writable.js:247:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.write (net.js:658:40)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.ondata (_stream_readable.js:555:20)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at emitOne (events.js:96:13)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.emit (events.js:188:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at readableAddChunk (_stream_readable.js:176:18)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Restarting browser...
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error executing phantom at /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: spawn /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs ENOENT
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at exports._errnoException (util.js:1022:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at onErrorNT (internal/child_process.js:359:16)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at process._tickCallback (internal/process/next_tick.js:98:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Module.runMain (module.js:606:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at run (bootstrap_node.js:394:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at startup (bootstrap_node.js:149:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at bootstrap_node.js:509:3
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: events.js:160
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: throw er; // Unhandled 'error' event
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: ^
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: This socket is closed
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._writeGeneric (net.js:680:19)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._write (net.js:731:8)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at doWrite (_stream_writable.js:334:12)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at writeOrBuffer (_stream_writable.js:320:5)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.Writable.write (_stream_writable.js:247:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.write (net.js:658:40)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.ondata (_stream_readable.js:555:20)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at emitOne (events.js:96:13)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.emit (events.js:188:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at readableAddChunk (_stream_readable.js:176:18)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Restarting browser...
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error executing phantom at /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: spawn /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs ENOENT
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at exports._errnoException (util.js:1022:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at onErrorNT (internal/child_process.js:359:16)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at process._tickCallback (internal/process/next_tick.js:98:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Module.runMain (module.js:606:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at run (bootstrap_node.js:394:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at startup (bootstrap_node.js:149:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at bootstrap_node.js:509:3
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: events.js:160
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: throw er; // Unhandled 'error' event
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: ^
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: This socket is closed
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._writeGeneric (net.js:680:19)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._write (net.js:731:8)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at doWrite (_stream_writable.js:334:12)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at writeOrBuffer (_stream_writable.js:320:5)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.Writable.write (_stream_writable.js:247:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.write (net.js:658:40)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.ondata (_stream_readable.js:555:20)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at emitOne (events.js:96:13)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.emit (events.js:188:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at readableAddChunk (_stream_readable.js:176:18)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Restarting browser...
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error executing phantom at /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: spawn /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs ENOENT
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at exports._errnoException (util.js:1022:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at onErrorNT (internal/child_process.js:359:16)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at process._tickCallback (internal/process/next_tick.js:98:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Module.runMain (module.js:606:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at run (bootstrap_node.js:394:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at startup (bootstrap_node.js:149:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at bootstrap_node.js:509:3
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: events.js:160
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: throw er; // Unhandled 'error' event
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: ^
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: This socket is closed
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._writeGeneric (net.js:680:19)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._write (net.js:731:8)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at doWrite (_stream_writable.js:334:12)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at writeOrBuffer (_stream_writable.js:320:5)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.Writable.write (_stream_writable.js:247:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.write (net.js:658:40)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.ondata (_stream_readable.js:555:20)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at emitOne (events.js:96:13)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.emit (events.js:188:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at readableAddChunk (_stream_readable.js:176:18)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Restarting browser...
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error executing phantom at /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: spawn /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs ENOENT
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at exports._errnoException (util.js:1022:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at onErrorNT (internal/child_process.js:359:16)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at process._tickCallback (internal/process/next_tick.js:98:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Module.runMain (module.js:606:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at run (bootstrap_node.js:394:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at startup (bootstrap_node.js:149:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at bootstrap_node.js:509:3
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: events.js:160
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: throw er; // Unhandled 'error' event
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: ^
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: This socket is closed
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._writeGeneric (net.js:680:19)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._write (net.js:731:8)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at doWrite (_stream_writable.js:334:12)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at writeOrBuffer (_stream_writable.js:320:5)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.Writable.write (_stream_writable.js:247:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.write (net.js:658:40)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.ondata (_stream_readable.js:555:20)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at emitOne (events.js:96:13)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.emit (events.js:188:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at readableAddChunk (_stream_readable.js:176:18)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Restarting browser...
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error executing phantom at /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: spawn /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs ENOENT
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at exports._errnoException (util.js:1022:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at onErrorNT (internal/child_process.js:359:16)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at process._tickCallback (internal/process/next_tick.js:98:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Module.runMain (module.js:606:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at run (bootstrap_node.js:394:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at startup (bootstrap_node.js:149:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at bootstrap_node.js:509:3
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: events.js:160
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: throw er; // Unhandled 'error' event
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: ^
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: This socket is closed
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._writeGeneric (net.js:680:19)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._write (net.js:731:8)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at doWrite (_stream_writable.js:334:12)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at writeOrBuffer (_stream_writable.js:320:5)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.Writable.write (_stream_writable.js:247:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.write (net.js:658:40)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.ondata (_stream_readable.js:555:20)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at emitOne (events.js:96:13)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.emit (events.js:188:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at readableAddChunk (_stream_readable.js:176:18)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Restarting browser...
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error executing phantom at /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: spawn /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs ENOENT
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at exports._errnoException (util.js:1022:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at onErrorNT (internal/child_process.js:359:16)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at process._tickCallback (internal/process/next_tick.js:98:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Module.runMain (module.js:606:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at run (bootstrap_node.js:394:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at startup (bootstrap_node.js:149:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at bootstrap_node.js:509:3
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: events.js:160
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: throw er; // Unhandled 'error' event
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: ^
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: This socket is closed
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._writeGeneric (net.js:680:19)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._write (net.js:731:8)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at doWrite (_stream_writable.js:334:12)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at writeOrBuffer (_stream_writable.js:320:5)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.Writable.write (_stream_writable.js:247:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.write (net.js:658:40)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.ondata (_stream_readable.js:555:20)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at emitOne (events.js:96:13)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.emit (events.js:188:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at readableAddChunk (_stream_readable.js:176:18)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Restarting browser...
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error executing phantom at /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: spawn /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs ENOENT
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at exports._errnoException (util.js:1022:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at onErrorNT (internal/child_process.js:359:16)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at process._tickCallback (internal/process/next_tick.js:98:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Module.runMain (module.js:606:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at run (bootstrap_node.js:394:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at startup (bootstrap_node.js:149:9)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at bootstrap_node.js:509:3
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: events.js:160
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: throw er; // Unhandled 'error' event
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: ^
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: This socket is closed
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._writeGeneric (net.js:680:19)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket._write (net.js:731:8)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at doWrite (_stream_writable.js:334:12)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at writeOrBuffer (_stream_writable.js:320:5)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.Writable.write (_stream_writable.js:247:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.write (net.js:658:40)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.ondata (_stream_readable.js:555:20)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at emitOne (events.js:96:13)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Socket.emit (events.js:188:7)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at readableAddChunk (_stream_readable.js:176:18)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Restarting browser...
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error executing phantom at /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: Error: spawn /usr/lib/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs ENOENT
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at exports._errnoException (util.js:1022:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at onErrorNT (internal/child_process.js:359:16)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at _combinedTickCallback (internal/process/next_tick.js:74:11)
Apr 27 08:56:11 cockpit-11.e2e.bos.redhat.com sh[77119]: at process._tickCallback (internal/process/next_tick.js:98:9)
^C[root@cockpit-11 cockpituous]# sudo systemctl stop cockpit-verify
```

File: verify/Dockerfile [hunk 1]
```diff
@@ -55,7 +54,6 @@ RUN echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
 RUN echo "config_opts['basedir'] = '/build/mock/'" >> /etc/mock/site-defaults.cfg
 RUN chmod -v u+s /usr/libexec/qemu-bridge-helper && echo 'allow cockpit1' >> /etc/qemu/bridge.conf
 
-ENV TEST_IMAGES_RHEL http://file.rdu.redhat.com/~swalter/images/
```
Comment (stefwalter) [inline verify/Dockerfile]: This has been removed on master.

**Comment by stefwalter** [comment]
Now that we're migrating away from phantomjs lets close this.

**Pull request closed without merging**